src/components/CaptureMatching/CaptureMatchingView.js

```diff
@@ -596,11 +665,8 @@ function CaptureMatchingView() {
                   </Typography>
                   <Typography variant="body1">
                     Joined at{' '}
-                    {format(
-                      growerAccount.first_registration_at ||
-                        growerAccount.created_at,
-                      'MM/dd/yyyy'
-                    )}
+                    {growerAccount.first_registration_at ||
```

Comment:
We need to format this to match other dates in the tool (ideally locale format).